utils/src/main/scala/za/co/absa/enceladus/utils/fs/FileSystemVersionUtils.scala

```diff
@@ -127,6 +128,54 @@ class FileSystemVersionUtils(conf: Configuration) {
     }
   }
 
+  /**
+    * Checks if a file is located in HDFS or in the local file system.
+    * If the file is in HDFS, it is copied to a temporary location.
+    *
+    * @param path A path to a file.  Can be either local or HDFS location.
+    * @return A path to a file in the local filesystem.
+    */
+  @throws[FileNotFoundException]
+  def getLocalFile(path: String): String = {
```

Comment:
I can't get over the fact that the method is called `getLocalFile` but is checking if the file is on HDFS and making the file local. Maybe `getLocalPathToFile` or something like that.

Reply:
I have the same problem.

Reply:
Struggling to come up with a good name. `getLocalPathToFile` seems decent. Do you agree on this, guys?

Reply:
Sure, good enough.